### PR TITLE
setup mongodb with example done

### PR DESCRIPTION
**Pull request by ManasesLovera**
Local mongodb setup done!

Tested and it's working, you must have mongodb shell or compose install to test.

Created Example Module with Schema > Service > Controller and imported in the application module.

Please let me know if there is anything that should be changed or fixed, I just followed the good nest practices for arquitecture but not for validation or dto stuff etc... cuz It's just an example.

### CODE REVIEW

**Review by Marlon-Vipe — APPROVED** [review]
All okay, approved

**Review by stealkiller06 — CHANGES_REQUESTED** [review]
Why didn't you use conventional commit? 
https://www.conventionalcommits.org/en/v1.0.0/

File: src/example/schemas/example.schema.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { HydratedDocument } from 'mongoose';
+
+type Object = {
```
[the remark this answers is not in the document]
Comment (ManasesLovera) [reply src/example/schemas/example.schema.ts:4]: it's empty because it's just an example

[the remark this answers is not in the document]
Comment (stealkiller06) [reply src/example/schemas/example.schema.ts:4]: don't like using Object as a name, can you change the name and not have it as an empty object?

https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Object

**Comment by ManasesLovera** [comment]
commit name updated

**Review by stealkiller06 — APPROVED** [review]

**Pull request merged**